quil-rs/src/instruction/frame.rs

```diff
@@ -30,12 +30,12 @@ impl Quil for AttributeValue {
     }
 }
 
-pub type FrameAttributes = HashMap<String, AttributeValue>;
+pub type FrameAttributes = BTreeMap<String, AttributeValue>;
 
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub struct FrameDefinition {
     pub identifier: FrameIdentifier,
-    pub attributes: HashMap<String, AttributeValue>,
+    pub attributes: FrameAttributes,
 }
```

Comment:
This is a breaking change.

Reply:
Added the necessary ! to the PR title to indicate a breaking change. Will make sure it's there when squashed to.